test/services/SubmissionServiceSpec.scala

```diff
@@ -41,42 +41,40 @@ import scala.concurrent.Future
 import scala.xml.{Node, Utility, XML}
 
 class SubmissionServiceSpec
-    extends WordSpec with MustMatchers with MockitoSugar with ScalaFutures with MovementsMetricsStub
-    with MetricsMatchers with BeforeAndAfterEach with UnitSpec {
+    extends UnitSpec with ScalaFutures with MovementsMetricsStub with MockCustomsExportsMovement with MetricsMatchers
+    with BeforeAndAfterEach with MockCustomsCacheService {
 
   implicit val defaultPatience: PatienceConfig =
     PatienceConfig(timeout = Span(5, Seconds), interval = Span(100, Millis))
 
   implicit val headerCarrierMock = mock[HeaderCarrier]
 
   val mockAuditService = mock[AuditService]
-  val customsCacheServiceMock = mock[CustomsCacheService]
-  val customsExportsMovementConnectorMock = mock[CustomsDeclareExportsMovementsConnector]
 
   val submissionService =
     new SubmissionService(
-      customsCacheServiceMock,
-      customsExportsMovementConnectorMock,
+      mockCustomsCacheService,
```

Comment:
remove mock prefix - as result of techtro

Reply:
It's old mock, renaming this will affect all places where we're using this mock. I prefer to leave it alone for now.